templates/single-product/add-to-cart/grouped.php

```diff
@@ -39,7 +39,7 @@
 				$post               = $post_object; // WPCS: override ok.
 				setup_postdata( $post );
 
-				echo '<tr id="product-' . esc_attr( $grouped_product->get_id() ) . '" class="woocommerce-grouped-product-list-item ' . esc_attr( implode( ' ', get_post_class( '', $grouped_product->get_id() ) ) ) . '">';
+				echo '<tr id="product-' . esc_attr( $grouped_product->get_id() ) . '" class="woocommerce-grouped-product-list-item ' . esc_attr( implode( ' ', wc_get_product_class( '', $grouped_product->get_id() ) ) ) . '">';
```

Comment:
Double escaping unless you address my other feedback.

Reply:
Yes, we need to avoid escaping here.

Reply:
Done with the last changes.